tcp.c

```diff
@@ -537,14 +555,15 @@ tcp_connect(char *server)
 void
 tcp_disconnect(void)
 {
-	if (g_ssl)
-	{
-		if (!g_network_error)
-			(void) SSL_shutdown(g_ssl);
-		SSL_free(g_ssl);
-		g_ssl = NULL;
-		SSL_CTX_free(g_ssl_ctx);
-		g_ssl_ctx = NULL;
+	int rv;
+	if (g_ssl_initialized) {
+		//rv = gnutls_bye(g_tls_session, GNUTLS_SHUT_RDWR);
```

Comment:
I guess this should have been removed now? :)

Reply:
Oh my God:)))))
Yes, it definitely should go now.
Will fix ASAP;)